docs/developer/typescript/01_gettingStarted/01_hello/index.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_label: Hello Ethereal
+---
+# Hello World from Ethereal Engine
+The quickstart tutorial helped us create a project and run the engine for the first time.  
+It automated a lot for us, so lets review what we have done so far.
+
+## Conceptual overview
+This is what we have done with the example's project:
+- We created an entity called `hello-world`
+- We gave the entity a primitive geometry component _(a sphere)_
+- We defined the position of the sphere in the scene
+
+## Technical overview
+This is what we have done with the example's source code:
+- We used the `ECS` pattern
+- We created an `Entity`
+- We added a few `Component`s to our Entity
```

Comment:
the `Component` with the `s` cut off is a little weird here. It's used elsewhere. Honestly I think it's fine to move the s into the code formating.

Reply:
The pattern is called `Entity Component System`, all singular, so this is done in order to use the word as a unique identifier that can be directly connected to its name when the brain goes through the process of absorbing the information.
The 's drawing a bit of attention might even be a good thing, since the goal of that type of formatting is aiding in memorization. :thinking:
Will do a pass to check if this is unnecessary in other places, though, just in case.

Reply:
`it'`s more distracting than you think and the demographic we are targeting can parse the root word from a plural word.

Reply:
i agree with daniel, we dont need to be so specific with the highlighting

Reply:
https://github.com/EtherealEngine/etherealengine-docs/pull/114/commits/96e2b8afb6d2991e512b3ffc2c5affbee7911628